source/MaterialXRenderGlsl/TextureBaker.cpp

```diff
@@ -97,6 +97,14 @@ FilePath generateOutMtlxFilename(string file, string srName)
     return out;
 }
 
+// Helper function to check if shader requires normals to be transformed from tangent space to world space
+bool connectsToNormalMapNode(OutputPtr output)
+{
+    ElementPtr normalMapNode = (output) ? output->getParent()->getChild(output->getAttribute("nodename")) : nullptr;
```

Comment:
In order to avoid this hard-coded "nodename" string, we can use the built-in method Output::getNodeName, which is provided by the base PortElement class.